src/BrainPass/BrainPass.sol

```diff
@@ -0,0 +1,402 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.13;
+
+import {ERC721} from "openzeppelin-contracts/contracts/token/ERC721/ERC721.sol";
+import {Counters} from "openzeppelin-contracts/contracts/utils/Counters.sol";
+import "openzeppelin-contracts/contracts/access/Ownable.sol";
+import "openzeppelin-contracts/contracts/security/Pausable.sol";
+import {Wiki} from "../../src/Wiki.sol";
+
+interface IERC20 {
+    function transferFrom(
+        address from,
+        address to,
+        uint256 amount
+    ) external returns (bool);
+
+    function balanceOf(address account) external view returns (uint256);
+
+    function transfer(address to, uint256 amount) external returns (bool);
+}
+
+/// @title BRAIN Pass NFT
+/// @author Oleanji
+/// @notice A pass for IQ Wiki Editors
+
+contract BrainPassCollectibles is ERC721, Pausable, Ownable {
```

Comment:
you can use ERC721Pausable and you dont need to define pause and other functions, less code